src/mongo/shell.ts

```diff
@@ -9,8 +9,11 @@ import { EventEmitter, window } from 'vscode';
 import { ext } from '../extensionVariables';
 import { IDisposable, toDisposable } from '../utils/vscodeUtils';
 
-export class Shell {
+// This is used at the end of each command we send to the console. When we get this string back,
+// we know we've reached the end of that command's result.
+const endOfDataBaseSentinel: string = '$EOD$';
```

Comment:
Just pointing out that DataBase still reads as Database. I suggest root for less confusion maybe? :P